### PR TITLE
kubecolor: init at 0.0.20

### PR DESCRIPTION
**Pull request by ivankovnatsky**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/cluster/kubecolor/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    owner = "hidetatz";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-bKHEp9AxH1CcObhNzD3BkNOdyWZu7JrEdsXpo49wEcI=";
+  };
+
+  vendorSha256 = "sha256-C1K7iEugA4HBLthcOI7EZ6H4YHW6el8X6FjVN1BeJR0=";
+
+  meta = with lib; {
+    description = "colorizes kubectl output";
```
Comment (legendofmiracles) [inline pkgs/applications/networking/cluster/kubecolor/default.nix:17]:
```suggestion
    description = "Colorizes kubectl output";
```

File: pkgs/applications/networking/cluster/kubecolor/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kubecolor";
+  version = "v0.0.20";
```
Comment (legendofmiracles) [inline pkgs/applications/networking/cluster/kubecolor/default.nix:5]:
```suggestion
  version = "0.0.20";
```

File: pkgs/applications/networking/cluster/kubecolor/default.nix [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+  src = fetchFromGitHub {
+    owner = "hidetatz";
+    repo = pname;
+    rev = version;
```
Comment (legendofmiracles) [inline pkgs/applications/networking/cluster/kubecolor/default.nix:10]:
```suggestion
    rev = "v${version}";
```

**Review by legendofmiracles — CHANGES_REQUESTED** [review]

**Comment by ivankovnatsky** [comment]
Let me squash it, please.

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 146742` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>kubecolor</li>
  </ul>
</details>

Doesn't print any output when running with any args, but it builds

**Review by legendofmiracles — APPROVED** [review]

**Pull request merged**